presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -0,0 +1,197 @@
+identified by BIGINT snapshot ids.  You can find the latest snapshot id for table
+foo by running the following command::
+
+    SELECT snapshot_id FROM 'foo$snapshots' ORDER BY committed_at DESC LIMIT 1
+
+A SQL procedure ``system.rollback_to_snapshot`` allows the caller to roll back
+the state of the table to a previous snapshot id::
+
+    CALL system.rollback_to_snapshot(schema_name, table_name, snapshot_id)
+
+Schema Evolution
+----------------
+
+Iceberg and the Iceberg connector support schema evolution, with safe
+column add, drop, reorder and rename operations, including in nested structures.
+Table partitioning can also be changed and the connector can still
+query data created before the partitioning change.
+
+Migrating Existing Tables
+-------------------------
+
+The connector can read from or write to Hive tables that have been migrated to Iceberg.
+Currently, there is no Presto support to migrate Hive tables to Presto, so you will
+need to use either the Iceberg API or Spark.
+
+System Tables and Columns
+-------------------------
+
+The connector supports ``table$partitions`` as a substitute for Hive's ``SHOW PARTITIONS``.
+The differences are that it returns some partition metrics for each partition value
+and you can also use it on unpartitioned tables.
+
+Iceberg supports ``$snapshot_id`` and ``$snapshot_timestamp_ms`` as hidden columns.
+These columns allow users to query an old version of the table. Think of this
+as a time-travel feature which lets you query your table's snapshot at a given time.
+
+Iceberg Table Properties
+------------------------
+
+================================================== ============================================================ ============
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ============
+``format``                                         Optionally specifies the format of table data files;         ORC
+                                                   either PARQUET or ORC.
+
+``partitioning``                                   Optionally specifies table partitioning.
+                                                   If a table is partitioned by columns c1 and c2, the
+                                                   partitioning property would be
+                                                   ``partitioning = ARRAY['c1', 'c2']``
+
+``location``                                       Optionally specifies the file system location URI for
+                                                   the table
+================================================== ============================================================ ============
+
+The table definition below specifies format PARQUET, partitioning by columns c1 and c2,
```

Comment:
Use code quotes for `PARQUET` or write it as the name Parquet. Code quotes around column names and location.

Reply:
Changed to Parquet and quotes added.